From 25860ff244205b7cc61fc5c8ed06bcbe9f12957f Mon Sep 17 00:00:00 2001
From: Ivan Romanov <drizt@land.ru>
Date: Fri, 14 Nov 2014 14:32:16 +0500
Subject: [PATCH 06/16] increased minimum cmake version

INCLUDE DESTINATION in install command was introduced only in CMake 2.8.12
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0f5a42c..7adacf1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -8,7 +8,7 @@ endif(NOT CMAKE_INSTALL_PREFIX)
 
 project(qca)
 
-cmake_minimum_required(VERSION 2.6.0)
+cmake_minimum_required(VERSION 2.8.12)
 
 set(QCA_LIB_MAJOR_VERSION "2")
 set(QCA_LIB_MINOR_VERSION "1")
-- 
1.9.3

